packages/components/checkbox/src/checkbox.tsx

```diff
@@ -38,11 +38,7 @@ const CheckboxRoot = chakra("label", {
 
 type CheckboxControlProps = Omit<HTMLChakraProps<"div">, keyof UseCheckboxProps>
 
-type BaseInputProps = Pick<
-  PropsOf<"input">,
-  "onBlur" | "checked" | "defaultChecked"
```

Comment:
`defaultChecked` is already a part of `UseCheckboxProps` so it doesn't have to be picked here